rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+    locations = cells_column_labels(columns = Levels)
+  ) %>%
+  tab_style(style = list(cell_text(color = 'gray', style = "italic")), 
+            locations = cells_body(columns = Requirement, 
+                                   rows = Requirement != "required")) %>%
+  tab_options(
+    row_group.background.color = "#FFEFDB",
+    table.font.size = "14px",
+    footnotes.marks = "letters")
+```
+
+
+## Additional considerations
+
+<!-- i.e. letter of data sharing law vs. the spirit of data sharing law -->
+The specifications should guide contributions in the "spirit of effective data sharing" and are most applicable when they lead to intended results, e.g. higher-impact shared data and reasonable balance of contributor/administrative effort given the expected value of the data shared. 
+<!-- would like to use "optimal" in place of "reasonable" someday; hard to optimize if expected values are unknown;
+are there any papers with quantitative data on what data types has actually been most reused?  -->
+But there can be debatable cases where a "required" specification is less applicable. 
+The community has anticipated issues like these and provided recommendations for some of them.
+<!-- we do want to minimize having to examine too many on a case-by-case basis, though -->
+
+### Imaging data
+
+Comments from the community noted two possible issues for imaging data. 
+
+A first question is whether imaging data should be shared when there are at most only a few images, which is of limited utility for re-analysis (e.g. with machine learning) usually requiring a large set of images.
+That is, imaging data may require a "critical mass".
+When only a few images are contributed, these are likely only representative images that might also appear in a publication.
+In this case, it may not be really "required" that these images have to be shared.
+
+A second case is for a study that may generate many images but can only process and fully annotate a subset.
+For example, there can be a large number of pathology images without any annotations of disease features by a pathologist, as these kinds of annotations are manually intensive.
+If all data are expected to be annotated -- and annotations are especially important for the usefulness of the images -- the question arises whether images outside the fully annotated subset should be shared. 
+A clear suggestion seems to be that sharing images without annotations is better than not sharing images -- for some images, it is possible for someone else to go back and add annotations.
+Note that basic, automatic annotations are still expected; using the example above, the images should still have an associated sample ID and tissue source.
+
+
+### Clinical data
+
+Clinical data can be extremely diverse and unstructured. 
```

Comment:
Maybe we can add the various types of clinical data that is possible, and then specify that the RFC asked for comments only on aggregated structured clinical metadata

Reply:
I think we can address this in a subsequent RFC, if needed.